### ISSUE TITLE
Heading permalinks

### ISSUE THREAD

**Issue by tophtucker** [BloombergMedia/whatiscode#46]

**Comment by jokeyrhyme** [comment]
Yeah, I'd love this, too. I'm working my way very slowly through this, and I keep forgetting where I'm up to. It would be so good to be able to make new bookmarks as I read.

**Comment by tophtucker** [comment]
@jokeyrhyme It's mad buggy, but have you found the time-spent-reading historgram on the far left (when you expand the table of contents) at all helpful? Anyway yeah we need this

**Comment by jokeyrhyme** [comment]
@tophtucker only noticed it because you pointed it out. :) I don't know how helpful it is, as I've been hopping between devices. /shrug
